docs/setup/security/notebook_authorization.md

```diff
@@ -81,3 +81,11 @@ For example, before executing a **Read** operation, it checks if the user and th
 Zeppelin executes a [REST API call](https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java) for the notebook permission information.
 In the backend side, Zeppelin gets the user information for the connection and allows the operation if the users and groups
 associated with the current user have at least one entity that belongs to owner entities for the notebook.
+
+## Permissions for folders
+Folder with permissions is way to unite notes with the same permissions and change the permissions together.
+
+Folder without permissions (with empty permissions) continues to work like common folder.
```

Comment:
+ 1 for this. Will be backward compatible

Reply:
@1ambda 
what do you mean? if downgrade version?

Reply:
I missed `+`, It means I like this behavior.